draft-ietf-mls-protocol.md

```diff
@@ -1327,9 +1327,11 @@ SignWithLabel(SignatureKey, Label, Content) =
 
 VerifyWithLabel(VerificationKey, Label, Content, SignatureValue) =
     Signature.Verify(VerificationKey, SignContent, SignatureValue)
+~~~
 
 Where SignContent is specified as:
 
+~~~ tls
```

Comment:
Anytime there is an equal sign and something that looks like a function definition, that is no longer TLS Presentation Language, but pseudocode.

I would move the definition of SignContent into a block like this before line 1334:
`~~~ tls`
`struct {`
`    opaque label<V>;`
`    opaque content<V>;`
`} SignContent;`

and then keep the content in the current block like this:

`    label = "MLS 1.0 " + Label;`
`    content = Content;`

Reply:
I see, thanks! I pushed a fixup in [360dbc7](https://github.com/mlswg/mls-protocol/pull/832/commits/360dbc7133c4923a4d7d29e921dc7587a013dfe3) and hope that I got your idea right.